app/controllers/administrations_controller.rb

```diff
@@ -49,13 +50,18 @@ def delete_question_concept
   end
 
   def add_question_concept
-    @c = Concept.find_by_name(params[:concept_name])
-    @q = LearningObject.find(params[:question_id])
-    unless @q.concepts.include? @c
-      @q.concepts << @c
+    if params[:concept_name].empty?
+      render nothing: true
       return
     end
 
+    @concept = Concept.find_by_name(params[:concept_name])
```

Comment:
Moze sa stat, ze dva kurzy budu mat rovnaky nazov konceptu (proceduralne programovanie (C) - rekurzia, funkcionalne programovanie (lisp) - rekurzia), takze by sa to malo vyhladat aj podla ID kurzu.